### PR TITLE
Add php7.3 to build matrix in allow failure mode.

### PR DESCRIPTION
**Pull request by markstory**
Lets see what happens 🤞

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ php:
   - 5.6
   - 7.1
   - 7.2
+  - 7.3
```
[the remark this answers is not in the document]
Comment (ADmad) [reply .travis.yml]: Actually seems we need to wait for https://github.com/travis-ci/travis-ci/issues/9717 to be resolved.

[the remark this answers is not in the document]
Comment (markstory) [reply .travis.yml]: Well I guess we'll have to wait then.

[the remark this answers is not in the document]
Comment (ravage84) [reply .travis.yml]: Regardless of when the related issue is resolved, what keeps us from adding nightly?

**Comment by ADmad** [comment]
Builds now running on PHP 7.3.0RC1. Had to tweak few tests for fix failures on other PHP version too since `xenial` distro is now used on travis.

Hopefully someone has use `hacktober` to fix the failures on PHP 7.3 :slightly_smiling_face:

**Pull request merged**